store/src/main/java/org/apache/rocketmq/store/CommitLog.java

```diff
@@ -1045,7 +1045,7 @@ public void run() {
 
             while (!this.isStopped()) {
                 try {
-                    this.waitForRunning(0);
+                    this.waitForRunning(1000);
```

Comment:
I also think `waitForRunning(0)` is too rigorous, but `waitForRunning(1000)` is adventurous while the `GroupCommitService` may miss some notify signals.

How about set a smaller wait time, like 10ms? what's your opinion ? @qinliujie